app/uk/gov/hmrc/alcoholdutyaccount/connectors/FinancialDataConnector.scala

```diff
@@ -95,8 +95,11 @@ class FinancialDataConnector @Inject() (config: AppConfig, implicit val httpClie
       "customerPaymentInformation" -> false.toString
     )
 
-  private def getOnlyOpenItemsFalseParameters(year: Int): Seq[(String, String)]               =
-    getBaseQueryParams(false) ++ Seq("dateFrom" -> s"$year-01-01", "dateTo" -> s"$year-12-31")
+  private def getOnlyOpenItemsFalseParameters: Seq[(String, String)] =
+    getBaseQueryParams(false) ++ Seq(
+      "dateFrom" -> "2024-10-31",
```

Comment:
Should we add these to app config so we have some degree of control over them?

Reply:
Sure, I can add them, I didn't add them earlier as I thought these were temporary changes and the pagination story would change this anyway..